check-swlib.sh

```diff
@@ -158,11 +154,31 @@ else
     echo "Found Ansible: $(type ansible-playbook)"
 fi
 
+declare -a CMD_ARRAY=()
```

Comment:
Following the pattern from line 179, I think we can write this a bit more concisely;  the intent here is to have it read like a normal command line, while still avoiding the security pitfalls of `eval`

declare -a CMD_ARRAY=("$ANSIBLE_PLAYBOOK" -i "$INVENTORY_FILE")